src/main/java/com/datalogics/pdf/samples/forms/FillForm.java

```diff
@@ -177,8 +177,6 @@ public static void fillAcroformFdf(final PDFDocument pdfDocument, final URL inpu
         final FDFService fdfService = new FDFService(pdfDocument);
         fdfService.importForm(fdfDocument);
 
-        // Run calculations on the AcroForm...
-        FormFieldService.getAcroFormFieldManager(pdfDocument).runCalculateScripts();
```

Comment:
I took out the calculation for XFDF.

Then, I put a "challenging" value into the XFDF file, and the test output fails in a way that seems to indicate that the calculations didn't happen.

/cc @datalogics-robl